azure-pipelines/integrationtests.yml

```diff
@@ -67,13 +67,13 @@ steps:
   - task: DotNetCoreCLI@2
     displayName: Building Integration Tests
     inputs:
-      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj'
+      projects: '**/Microsoft.SqlTools.ServiceLayer.IntegrationTests.csproj, **/Microsoft.SqlTools.SqlCore.IntegrationTests.csproj'

```

Comment:
This isn't comma delimited - see https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/file-matching-patterns?view=azure-devops for details. You could probably just do `**/*.IntegrationTests.csproj` to make it easier to update in the future